### PR TITLE
lsof: update to 4.98.0

### PR DESCRIPTION
**Pull request by subnut**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

**Comment by vincele** [comment]
Briefly tested on x86_64-musl, LGTM

File: srcpkgs/lsof/patches/fix-epoll-test.patch [hunk 1]
```diff
@@ -0,0 +1,85 @@
+This patch has been upstreamed already, and should be
```
Comment (classabbyamp) [inline srcpkgs/lsof/patches/fix-epoll-test.patch:1]: add a link to it please

Comment (vincele) [reply srcpkgs/lsof/patches/fix-epoll-test.patch:1]: I think this is it:
https://github.com/void-linux/void-packages/pull/41909/files#diff-122915c48ac80e5e28054e63f87f41571facf87faf46b0700d0d9fa9bb1dc9c7

Comment (subnut) [reply srcpkgs/lsof/patches/fix-epoll-test.patch:1]: @classabbyamp Here -
https://github.com/lsof-org/lsof/commit/5ca335ff9ae6510cba0a94a62fbaa46709f7cb16

Do I need to mention it in the patch?

Comment (classabbyamp) [reply srcpkgs/lsof/patches/fix-epoll-test.patch:1]: yes please

**Pull request merged**